pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -778,17 +771,6 @@ func getRunNamesFromChildRefs(childRefs []v1beta1.ChildStatusReference, ptName s
 	return runNames
 }
 
-// resolvePipelineTaskResources matches PipelineResources referenced by pt inputs and outputs with the
-// providedResources and returns an instance of ResolvedTask.
-func resolvePipelineTaskResources(_ v1beta1.PipelineTask, ts *v1beta1.TaskSpec, taskName string, kind v1beta1.TaskKind) (*resources.ResolvedTask, error) {
```

Comment:
Thanks! I have the same change in trusted resources PR, I think this is a great improvement.